crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -74,10 +71,20 @@ pub struct Options {
     #[serde(default)]
     pub development: Option<bool>,
 
-    /// TODO: Remove this field.
+    // @babel/plugin-transform-react-jsx: Since "useBuiltIns" is removed in Babel 8, you can remove
+    // it from the config.
+    #[deprecated(
+        note = r#"This is always true. Use swc_ecma_transforms_compat to downlevel JavaScript syntax."#
```

Comment:
Wait a minute, I should update the comments/

Reply:
@kdy1 done.

Reply:
Cool, thanks!